src/Aeon/Emulator/Video/Rendering/GraphicsPresenter8.cs

```diff
@@ -0,0 +1,35 @@
+namespace Aeon.Emulator.Video.Rendering
+{
+    /// <summary>
+    /// Renders 8-bit graphics to a bitmap.
+    /// </summary>
+    public sealed class GraphicsPresenter8 : Presenter
+    {
+        /// <summary>
+        /// Initializes a new instance of the GraphicsPresenter8 class.
+        /// </summary>
+        /// <param name="videoMode">VideoMode instance describing the video mode.</param>
+        public GraphicsPresenter8(VideoMode videoMode) : base(videoMode)
+        {
+        }
+
+        /// <summary>
+        /// Updates the bitmap to match the current state of the video RAM.
+        /// </summary>
+        protected override void DrawFrame(nint destination)
+        {
+            uint totalPixels = (uint)VideoMode.Width * (uint)VideoMode.Height;
+            ReadOnlySpan<uint> palette = VideoMode.Palette;
+
+            unsafe
+            {
+                byte* srcPtr = (byte*)VideoMode.VideoRam.ToPointer() + (uint)VideoMode.StartOffset;
```

Comment:
All Presenters: Try to use Spans for source pointers. Only pointers for the destination buffer from Avalonia (since this can't be avoided).